.env.example

```diff
@@ -0,0 +1,8 @@
+API_KEY="<API_KEY>"
+VITEST_SEPOLIA_FORK_URL="https://eth-sepolia.g.alchemy.com/v2/<API_KEY>"
+PAYMASTER_POLICY_ID="<YOUR_PAYMASTER_POLICY_ID>"
+OPENAI_API_KEY="" # leave empty to skip docs generation step or if you are an external contributor
+VITEST_OPT_SEPOLIA_FORK_URL="https://opt-sepolia.g.alchemy.com/v2/<API_KEY>"
+ALCHEMY_API_URL="https://api.g.alchemy.com/"
+ALCHEMY_RPC_URL="https://eth-sepolia.g.alchemy.com/v2/<API_KEY>/"
+ALCHEMY_RPC_URL_ODYSSEY="https://eth-sepolia.g.alchemy.com/v2/<API_KEY>/"
```

Comment:
are these still being used? for example, i don't see `ALCHEMY_RPC_URL_ODYSSEY` or `VITEST_OPT_SEPOLIA_FORK_URL` in the codebase. can you do another pass through and comment what each one is used for?

Reply:
should this live here? if most of these variables are only used in the demo app, i would expect this to be in the demo app directory.

i personally do not even use a `.env` file at the root of this monorepo.